vnext/Microsoft.ReactNative.ComponentTests/Microsoft.ReactNative.ComponentTests.vcxproj

```diff
@@ -158,6 +164,13 @@
     </ClCompile>
     <ClInclude Include="CommonReaderTest.h" />
     <ClInclude Include="pch/pch.h" />
+    <ClCompile Include="$(JSI_SourcePath)\jsi\test\testlib.cpp">
```

Comment:
IIRC we have a separate source copy of JSI tests in our repo, from before they were published. Consider deleting those and changing them to include from JSI SourcePath like this is doing. 

Related to that, I'm not familiar enough with existing project structure, but could we reconcile having multiple projects running JSI tests?

Reply:
I want to address it in future. The issue is that the old tests that we run for the Desktop environment are also used for V8 and to replace them with the stock tests there will require fixing the V8 JSI. It seems to be out of scope of this PR.

We will reconcile the tests, but we should do it in future after Julio removes the JSI projects. I think it will be done as we go towards the single RN core DLL.

---
In reply to: [470192372](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470192372) [](ancestors = 470192372)